website/integrations/services/aws/index.md

```diff
@@ -75,3 +82,94 @@ To use the user's username, use this snippet
 ```python
 return user.username
 ```
+
+# Method 2: IAM Identity Center
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `authentik.company` is the FQDN of the authentik install.
+
+Additional Preparation:
+
+-   A certificate to sign SAML assertions is required. You can use authentik's default certificate, or provide/generate one yourself.
+-   You may pre-create an AWS application
+
+## Procedure
+
+In AWS:
+
+-   In AWS Navigate to: `IAM Identity Center -> Settings -> Identity Source (tab)`
+-   On the right hand side click `Actions -> Change identity source`
+-   Select `External Identity Provider`
+-   Under `Service Provider metadata` it'll allow you to download a metadata file. Download this file.
+
+Now go to your authentik instance, and perform the following steps.
+
+-   Under _Providers_ create a new _SAML Provider from metadata_. Give it a name, and upload the metadata file AWS gave you.
+-   Click _Next_. Give it a name, and close.
+-   If you haven't done so yet, create an application for AWS and connect the Provider to it.
```

Comment:
```suggestion
-   If you haven't done so yet, create an application for AWS and connect the provider to it.
```

Reply:
ideally we would be able to link to a procedural doc on how to create an app... but we will add that later.